src/main/java/com/neuronrobotics/bowlerstudio/scripting/SshTransportConfigCallback.java

```diff
@@ -1,27 +1,15 @@
 package com.neuronrobotics.bowlerstudio.scripting;
 
 import org.eclipse.jgit.api.TransportConfigCallback;
-import org.eclipse.jgit.transport.JschConfigSessionFactory;
-import org.eclipse.jgit.transport.OpenSshConfig;
-import org.eclipse.jgit.transport.SshSessionFactory;
 import org.eclipse.jgit.transport.SshTransport;
 import org.eclipse.jgit.transport.Transport;
 
-import com.jcraft.jsch.Session;
-
 public class SshTransportConfigCallback implements TransportConfigCallback {
+    org.eclipse.jgit.transport.sshd.SshdSessionFactory sshSessionFactory = new org.eclipse.jgit.transport.sshd.SshdSessionFactory();
```

Comment:
This line was the specific change that resolved https://github.com/CommonWealthRobotics/BowlerStudio/issues/339